tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

```diff
@@ -39,6 +38,7 @@
   private static final String DEFAULT_NO_SIGNALS_TIMEOUT_MILLIS_ENV = "DEFAULT_NO_SIGNALS_TIMEOUT_MILLIS";
 
   private final long defaultTimeoutMillis;
+  private final long defaultPollTimeoutMillis;
```

Comment:
I wonder if the name shouldn't be `defaultErrorPollTimeoutMillis` as it does _not_ really touch any of the next/completion queue things, as those always used the ABQ: `      Optional<T> value = abq.poll(timeoutMillis, TimeUnit.MILLISECONDS);` so there's no change to that behavior by the introduction of this new param... WDYT @Scottmitch ?

Reply:
The rational was the `error` handling is an implementation detail and this polling can be used for all things that would otherwise have to block/wait the full `defaultTimeoutMillis` duration (e.g. if there is no async pre-emption). Although it is currently used for expected error timeouts due to internal implementation details, if the implementation was to change or be extended I think it is useful to have a "poll interval" and "upper bound" for these cases. However I'll defer to you if you want it more specialized.

Reply:
Ok that's a good point, let's keep the general wording for "whenever would have to poll" 👍